authentication/README.md

```diff
@@ -11,6 +11,9 @@ In Unix-style shells and an environment file called `.env`, the environment is s
 source .env
 ```
 
+### Dotenv Environment
+Some sample applications take advantage of libraries that read environment files. This is more flexible than reading the values of the variables that have been set in the shell. These environment files, often called `.env` as well. The contents of the files will differ however. No shell commands may be executed in these files. Nor should variables be exported.
```

Comment:
It's a bit cryptic and assumes the reader has a mental model of environment variables as it relates to shells and sub-processes of that shell.

The gist is that sample applications can be provided environment-specific configuration (such as customer-specific values we cannot add to a public repository because they're secret or generally because they are customer-specific).

That config can be 'exported' to applications that are started from a shell. This is the rigid way of doing things, i.e. rely on the OS to expose these variables to the sample application (sub-process). A more flexible way (using the libraries I refer to) is to bypass telling the OS/shell about this config and simply keep it in files that the application (through the use of libraries like `dotenv`) reads directly.
The downside is that you have less expressive power in these files, but the upside (which is huge) is that you have much more granular control over the values provided for variables. In particular, it'll allow you to specify per-application config, which is essential when your run applications from a server like JupyterLab which will host many such applications where it's impractical to attempt to provide configuration for all available applications through the OS level (that started the JupyterLabs server).
Clear as mud?

Reply:
Based on our convo earlier, I think this is a closer representation of what we want to say (which is that the API samples use Dotenv and why that's a good thing, plus best practices for .env files):
Some sample applications use libraries that read environment files. Libraries that read environment files are more flexible than reading the variable values set in the shell. This repository uses Dotenv to load environment files (`.env`). The contents of each `.env` file differs; you cannot execute shell commands in `.env` files and should not export variables from any `.env` files.

Reply:
Fine with me. Pls approve when done